com.unity.visualeffectgraph/Editor/GraphView/VFXComponentBoard.cs

```diff
@@ -607,6 +607,9 @@ void Update()
                 path = m_AttachedComponent.gameObject.scene.name + " : " + path;
             }
 
+            if (m_Subtitle.text != path)
```

Comment:
These two lines were removed (by mistake) in the PR referenced in the description

Reply:
👍